#!/bin/sh /usr/share/dpatch/dpatch-run
## add_debian_readme.dpatch by  <khalid@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch adds a README.debian file to explain debian specifc
## DP: packaging details

@DPATCH@
diff -urN kexec-tools-2.0.0/README.debian kexec-tools-2.0.0.debian/README.debian
--- kexec-tools-2.0.0/README.debian	1969-12-31 17:00:00.000000000 -0700
+++ kexec-tools-2.0.0.debian/README.debian	2009-04-16 11:40:40.000000000 -0600
@@ -0,0 +1,11 @@
+This kexec-tools package was built from kexec-tools git repository at 
+<git://git.kernel.org/pub/scm/linux/kernel/git/horms/kexec-tools.git>. 
+
+Installing this package also installs two init scripts under /etc/init.d - 
+kexec-load and kexec. kexec-load is called upon reboot to load a new kexec
+kernel if kexec is enabled in /etc/default/kexec. kexec script is called 
+just before reboot script. It checks if kexec is enabled and a
+kexec'able kernel is loaded. If yes, it kexecs a new kernel instead of
+reboot. As a result, kexec is automatically enabled upon installation of 
+kexec-tools package if kexec is enabled in /etc/default/kexec through debconf
+option.
